easybuild/easyconfigs/p/Pilon/Pilon-1.22-Java-1.8.0_92.eb

```diff
@@ -0,0 +1,27 @@
+easyblock = 'JAR'
+
+name = 'Pilon'
+version = '1.22'
+versionsuffix = '-Java-%(javaver)s'
+
+homepage = 'https://github.com/broadinstitute/pilon'
+description = "Pilon is an automated genome assembly improvement and variant detection tool"
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+source_urls = ['https://github.com/broadinstitute/pilon/releases/download/v%(version)s/']
+sources = ['%(namelower)s-%(version)s.jar']
+
+dependencies = [('Java', '1.8.0_92')]
+
+sanity_check_paths = {
+    'files': ["%(namelower)s-%(version)s.jar"],
+    'dirs': [],
+}
+
+postinstallcmds = ['cd %(installdir)s && ln -s %(namelower)s-%(version)s.jar pilon.jar']
+
+modloadmsg = "To execute Pilon run: java -Xmx8G -jar $EBROOTPILON/pilon.jar\n"
+modloadmsg += "\rAdjust the memory value according to the size of your input files\n\r"
```

Comment:
this format is because of this issue https://github.com/easybuilders/easybuild-framework/issues/2405